CHANGELOG.md

```diff
@@ -1,3 +1,197 @@
+* [Allow clients to calculate when a view appears/disappears. (#7841)](https://github.com/material-components/material-components-ios/commit/3a0c1ee2e8bd2257894ab696d4759ab67e390993) (Robert Moore)
+* [Combine "default" unit tests. (#7822)](https://github.com/material-components/material-components-ios/commit/bc008a2ae6fc27a49370dbfb6abed0c0660c1467) (Robert Moore)
+* [Don't handle `contentInset` directly in layout. (#7816)](https://github.com/material-components/material-components-ios/commit/b67ae677623ac314f037e5c35f38e40117e84de6) (Robert Moore)
+* [Fix crash when assigning `nil` for KVO'd title. (#7806)](https://github.com/material-components/material-components-ios/commit/fffcb4a5a006b34a157fcac4c9cc89d630e0976a) (Robert Moore)
+* [Fix segmented control position. (#7898)](https://github.com/material-components/material-components-ios/commit/640220d66d277047113430c566ea5b161692e959) (Robert Moore)
+* [Fix title-only example. (#7897)](https://github.com/material-components/material-components-ios/commit/f2b258014408fa90a3d60b2e6e3dc027ae47bfa9) (Robert Moore)
+* [Minor refactoring of KVO code. (#7825)](https://github.com/material-components/material-components-ios/commit/bd61834803047560d3f6095ffa72c4b52cba5934) (Robert Moore)
+* [Retain custom view in example. (#7901)](https://github.com/material-components/material-components-ios/commit/4d99e843059129e84ef9061603e092f8b537faaa) (Robert Moore)
+
+### TextFields
+
+* [Textfields dynamic color support (#7889)](https://github.com/material-components/material-components-ios/commit/b2f2a5d77fead5f1164b2884bf540f3295f48927) (Andrew Overton)
+
+### private/Snapshot
+
+* [added ios13 support for snapshots (#7861)](https://github.com/material-components/material-components-ios/commit/48f78118a7d268c92521bad12b118238de74a3fb) (Yarden Eitan)
+
+### schemes/Color
+
+* [Add a default color scheme for Dark mode. (#7831)](https://github.com/material-components/material-components-ios/commit/666b17a34646ea9dde490e1a22c65ab806c6d5bc) (Wenyu Zhang)
+* [Add dynamic color scheme MDCColorSchemeDefaultsMaterial201907. (#7848)](https://github.com/material-components/material-components-ios/commit/6f83e12273d3739dba06f48a1cac3d09865b3cb2) (Wenyu Zhang)
+* [Use UIColor method instead of static method to generate (#7894)](https://github.com/material-components/material-components-ios/commit/0175300afc821396a31b5cd7d0bd638b9ce58dac) (Wenyu Zhang)
+* [add two properties for elevation overlay color in color (#7924)](https://github.com/material-components/material-components-ios/commit/914e9a37fb0ef1e4a3663126bb61997c3600920c) (Wenyu Zhang)
+
+## Multi-component changes
+
+* [Add UIColor+MaterialDynamic to support dynamic color. (#7860)](https://github.com/material-components/material-components-ios/commit/a65f36d4beaa7d8859c93a046967270375387d13) (Wenyu Zhang)
```

Comment:
Please classify this as `private/Color` since this is a new component and not multi-component.

Reply:
Addressed in 3d57d80f41507c3269e53a9ded24f489eb3f80a0